### PR TITLE
Add aclapi definitions

### PR DESCRIPTION
**Pull request by skdltmxn**

### CODE REVIEW

**Comment by skdltmxn** [comment]
I don't know why it's failing

**Comment by Susurrus** [comment]
rustup failed to install Rust 1.14, so that command errored out. Then during the testing rust 1.14+ didn't exist so trying to compile using them failed for all of them. Basically just re-push this after amending the commit and see if it works again. Otherwise @retep998 can rerun this from the Travis interface.

**Comment by retep998** [comment]
I'll rerun this from the travis interface once I get confirmation from someone that the issue is fixed.

**Comment by Susurrus** [comment]
Who is that someone you're waiting for? As far as I could tell this was a sporadic network failure so the tests should just be rerun to confirm this.

**Comment by Susurrus** [comment]
Actually, we in the nix crate are getting similar errors with Rust 1.14. I haven't seen a bug posted about it however.

**Comment by retep998** [comment]
I was waiting for https://github.com/rust-lang/rust/issues/45303#issuecomment-336718037

File: src/um/accctrl.rs [hunk 1]
```diff
@@ -0,0 +1,368 @@
+// Copyright © 2016-2017 winapi-rs developers
```
Comment (retep998) [inline src/um/accctrl.rs]: Since this is a new file the copyright year should be just 2017.

**Comment by skdltmxn** [comment]
Fixed!

**Comment by skdltmxn** [comment]
It looks like #447 already contains these updates.
Should I close this PR?

**Comment by retep998** [comment]
If this PR was completely superseded by another merged PR, then probably.

**Pull request closed without merging**